tests/model_forms/tests.py

```diff
@@ -2177,6 +2178,15 @@ def test_model_multiple_choice_field(self):
         with self.assertRaises(ValidationError):
             f.clean([c6.id])
 
+    def test_model_multiple_choice_invalid_pk_value_error_messages(self):
```

Comment:
It appears to me that this test duplicates the other two. What's your thinking?

Reply:
It looks like I uploaded the wrong version. I'll update it with the test path Sarah suggested and submit it again.